specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json

```
@@ -205,6 +205,89 @@
         "type": "object"
       }
     },
+    "DelimitedTextReadSetting": {
+      "description": "Delimited text read settings.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/FormatReadSetting"
+        }
+      ],
+      "properties": {
+        "skipLineCount": {
+          "type": "object",
+          "description": "Indicates the number of non-empty rows to skip when reading data from input files. Type: integer (or Expression with resultType integer)."
+        },
+        "addtionalNullValues": {
```

addtional -> additional
```suggestion
        "additionalNullValues": {
```

---

additionalNullValues is a private peroperty. Please remove it from public swagger. Thanks.

---

ok